### PR TITLE
App to ts

### PR DESCRIPTION
**Pull request by Idokah**

### CODE REVIEW

File: apps/velo-external-db/src/app.ts [hunk 1]
```diff
@@ -1,16 +1,18 @@
-const express = require('express')
-const { create, readCommonConfig } = require('@wix-velo/external-db-config')
-const { ExternalDbRouter } = require('@wix-velo/velo-external-db-core')
-const { engineConnectorFor } = require('./storage/factory')
+import express = require('express')
```
[the remark this answers is not in the document]
Comment (Idokah) [reply apps/velo-external-db/src/app.ts:1]: Yes, 
they export it with: 
`export = express;`

File: apps/velo-external-db/src/app.ts [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
-let server, _schemaProvider, _cleanup
-let externalDbRouter
+let server: Server, _schemaProvider: ISchemaProvider, _cleanup: ConnectionCleanUp
+let externalDbRouter: ExternalDbRouter
 
 
 
-
-const initConnector = async(hooks) => { 
+const initConnector = async (hooks?: { dataHooks: DataHooks, schemaHooks: SchemaHooks }) => {
```
[the remark this answers is not in the document]
Comment (Idokah) [reply apps/velo-external-db/src/app.ts:15]: What's the purpose? 
make a type: `Hooks: {dataHooks: DataHooks,  schemaHooks:SchemaHooks})` ?

File: apps/velo-external-db/src/global.d.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+import 'jest-extended'
```
[the remark this answers is not in the document]
Comment (Idokah) [reply apps/velo-external-db/src/global.d.ts:1]: You see everything god dam

File: apps/velo-external-db/src/app.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (MXPOL) [reply apps/velo-external-db/src/app.ts:15]: yes  :)

**Review by MXPOL — APPROVED** [review]

File: apps/velo-external-db/src/app.ts [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
-const { create, readCommonConfig } = require('@wix-velo/external-db-config')
-const { ExternalDbRouter } = require('@wix-velo/velo-external-db-core')
-const { engineConnectorFor } = require('./storage/factory')
+import express = require('express')
+import { create, readCommonConfig } from '@wix-velo/external-db-config'
+import { ExternalDbRouter } from '@wix-velo/velo-external-db-core'
+import { engineConnectorFor } from './storage/factory'
+import { Server } from 'http'
+import { ConnectionCleanUp, ISchemaProvider } from '@wix-velo/velo-external-db-types'
+import { Hooks } from 'libs/velo-external-db-core/src/types'
```
Comment (MXPOL) [inline apps/velo-external-db/src/app.ts:7]: @Idokah importing from another package via path and not via the package is really bad idea .....

**Pull request merged**